src/js/themes/base.d.ts

```diff
@@ -31,6 +31,7 @@ import { Box } from '../components/Box';
 import { Text, TextProps } from '../components/Text';
 import { LayerPositionType } from '../components/Layer';
 import { DropProps } from '../components/Drop';
+import { string } from 'prop-types';
```

Comment:
Don't think we need this, it's not being used

Reply:
Codesandbox was being too helpful ;)

Reply:
Removed.